setup.cfg

```diff
@@ -23,7 +23,7 @@ packages = find:
 install_requires =
     auditree-framework>=1.2.3
     auditree-harvest>=1.0.0
-    pyyaml<5.4
+    pyyaml>=5.4
```

Comment:
Probably `5.4.1`, as that is the version which fixed the vulnerability, but consumers could decide that on their own with a `constraints.txt` file.